homeassistant/components/todoist/calendar.py

```diff
@@ -194,14 +180,16 @@ async def handle_new_task(call: ServiceCall) -> None:
         data: dict[str, Any] = {"project_id": project_id}
 
         if task_labels := call.data.get(LABELS):
-            data["label_ids"] = [
-                label_id_lookup[label.lower()] for label in task_labels
-            ]
+            data["labels"] = task_labels
 
         if ASSIGNEE in call.data:
+            collaborators = await api.get_collaborators(project_id)
+            collaborator_id_lookup = {
+                collab.name.lower(): collab.id for collab in collaborators
+            }
             task_assignee = call.data[ASSIGNEE].lower()
             if task_assignee in collaborator_id_lookup:
-                data["assignee"] = collaborator_id_lookup[task_assignee]
+                data["assignee_id"] = collaborator_id_lookup[task_assignee]
```

Comment:
This was intentional, the param was incorrect per the docs: https://developer.todoist.com/rest/v2/?shell#create-a-new-task